packages/dd-trace/src/ci-visibility/exporters/agentless/request.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const https = require('https')
+const http = require('http')
+const log = require('../../../log')
+
+function retriableRequest (data, options, callback) {
```

Comment:
Can we reuse the client from the agent exporter, or at least parts of it? Right now it looks like it was mostly copy pasted.

Reply:
I'll give it another try but I'm not sure it's going to be much of an improvement: `data` here is not a list (like in agent), we don't use `http.Agent` (yet), we don't set the container id, and the retry here happens always, not only because of a socket connection close. These little differences make it a bit difficult to extract in a common function without having to include a lot of conditional clauses

Reply:
About the above points, I think we could normalize everything and use `http.Agent` everywhere, set the container ID even if it's not used yet and let the retry happen always for both exporters. Basically, both use HTTP and should have similar sets of rules for the connection.

Reply:
I'd really rather not use the `http.Agent` until I've tested the version without it (I've had problems with it), but that can be handled in a single conditional clause.

Reply:
I've done the whole rewrite in https://github.com/DataDog/dd-trace-js/pull/1880/commits/301035d551b5c9c51006add92107e1935b76ec4b, so there's a single `request` module. I've also moved the test files